### DISCUSSION TITLE
Your Favorite Language!!(Python, Japanese etc...)

### DISCUSSION THREAD

**Discussion by kaircx** [NAIST-SE/AttractingContributors#857] [General]
[リポジトリ](https://github.com/kaircx/AttractingContributors-favorite-language/)

私たちの言語リポジトリへようこそ！ここは、日本語やドイツ語のような話し言葉から、CやPythonのようなプログラミング言語まで、あらゆる形の言語を祝う場所です。母国語であろうと、学んでいる外国語であろうと、コードを書くのが好きなプログラミング言語であろうと、皆さんがお気に入りの言語を共有・寄稿することを歓迎します。一緒に、言語の美しさと多様性を称えましょう！
___
[Repository](https://github.com/kaircx/AttractingContributors-favorite-language/)

Welcome to our Language Repository! This place is a celebration of languages in all forms, from spoken languages like Japanese and German, to programming languages like C and Python. We invite everyone to contribute and share their favorite languages, be it your mother tongue, a foreign language you're learning, or a programming language you love to code in. Join us and let's appreciate the beauty and diversity of languages together!

**Comment by oto10** [comment]
I added my favorite language to your list.
Could you please add your favorite music to my README.md?
https://github.com/oto10/AttractingContributors-your-favorite-music
Thank you in advance.

**Comment by k-mizoguch1** [comment]
plz https://github.com/kzu000/Food-what-do-you-like
